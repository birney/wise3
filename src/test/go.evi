exon 1404 1493
exon 1789 1947
exon 2120 2294
exon 2388 2461
