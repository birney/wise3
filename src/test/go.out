Gene 1
Gene 1401 2461 
  Exon 1401 1493 phase 0
  Exon 1789 1947 phase 0
  Exon 2120 2294 phase 0
  Exon 2388 2461 phase 1
//
>HSHNRNPA.[1401:2461].sp.tr
RKLFIGGLSFETTDESLRSHFEQWGTLTDCVVMRDPNTKRSRGFGFVTYATVEEVDAAMN
ARPHKVDGRVVEPKRAVSREVSGFKIFVGGIKEDTEEHHLRDYFEQYGKIEVIEIMTDRG
SGKKRGFAFVTFDDHDSVDKIVIQKYHTVNGHNCEVRKALSKQEMAS
//
Gene
  utr5 1400 1400
  cds 1401 1493 phase 0
  cds 1789 1947 phase 0
  cds 2120 2294 phase 0
  cds 2388 2461 phase 1
End
//
